config/webpack.js

```diff
@@ -492,10 +492,17 @@ module.exports = (env) => {
         fail_build: true,
       };
 
-      if (process.env.TRAVIS === 'true') {
+      if (process.env.TRAVIS === 'true' || process.env.CI === 'true') {
```

Comment:
Can just check `process.env.CI === 'true'`, travis sets that too

Reply:
Will do